src/system/BlocksModule/Controller/BlockController.php

```diff
@@ -110,7 +110,7 @@ public function editAction(Request $request, BlockEntity $blockEntity = null)
             $renderedPropertiesForm = $this->formContentModify($request, $blockEntity);
         }
 
-        if ($form->isSubmitted() and $form->get('save')->isClicked() and $form->isValid()) {
+        if ($form->isSubmitted() && $form->get('save')->isClicked() && $form->isValid()) {
```

Comment:
From SensioLabsInsights:

The `and` operator does not have the same precedence as `&&`. This could lead to unexpected behavior, use `&&` instead.